src/adapters/scene.js

```diff
@@ -39,6 +40,8 @@ Scene.prototype.initMapBox = function () {
   })
   this.popup.init(this.mb)
 
+  setRTLTextPlugin('/statics/javascript/mapbox-gl-rtl-text.js');
```

Comment:
Absolute paths need to be prefixed with `baseUrl`, instead of `/`.
See line 144 for example.